src/Discovery/DiscoveryActionBar/components/OpenFillRequestFormButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { FileTextOutlined } from '@ant-design/icons';
+import { Popover, Button } from 'antd';
+
+const OpenFillRequestFormButton = ({
+    props,
+  }) => (
+  props.config.features.exportToWorkspace?.openFillRequestForm && (
+    <Popover
+      className='discovery-popover'
+      arrowPointAtCenter
+      title={(
+        <React.Fragment>
+          &nbsp;
+          <a target='_blank' rel='noreferrer' href='https://pandemicresponsecommons.org'>
+            {'Pandemic Response Commons Website'}
+          </a>.
+        </React.Fragment>
+      )}
+      content={(
+        <span className='discovery-popover__text'>
+          After filling the request form, after your search selection, you would get an approval after which you can use the Gen3 Client
+          to download the data from the selected studies to your local computer.
+        </span>
+      )}
+    >
+      <Button
+        onClick={props.config.features.exportToWorkspace?.fillRequestFormURL ? () => {
+          let combinedIds = '';
+          props.discovery.selectedResources.forEach((item) => {
+            if (combinedIds !== '') {
+              combinedIds += ',';
+            }
+            combinedIds += item._medical_sample_id;
+          });
```

Comment:
L29-35 can be replaced by
```
const combinedIds = props.discovery.selectedResources.map((item)=>(item._medical_sample_id)).join(',');
```

Reply:
Change done, I will commit together once all of this is resolved